degica/komoju-woocommerce
# Move non-payment-method-specific settings into dedicated "KOMOJU" tab

My ultimate goal right now is to make each KOMOJU payment method a separate WC gateway.

Currently, there is just one "KOMOJU" gateway that contains options for each actual payment method. This causes a handful of problems, including
1. poor interop with other plugins (e.g. payment gateway filtering plugins) and
2. confused customers, not knowing what a KOMOJU is.

This PR doesn't quite achieve the end-goal of one gateway per payment method. This PR just moves settings that would apply to all payment methods into a dedicated KOMOJU settings tab. 
![image](https://user-images.githubusercontent.com/2793160/169475491-a731783e-12a0-4ed8-8349-ce5b14e03769.png)

## How to test

1. Follow the [dev setup guide](https://github.com/degica/komoju-woocommerce/blob/master/docs/dev_setup.md) if you don't already have a WC store set up
2. Make sure you're using this branch
3. Go to your store settings
4. See the "Komoju" tag (in a future PR, we should change all "Komoju" texts to "KOMOJU")
5. Confirm that you can update your api key and other things from here
